Lesson1/Task1/Program.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+
+
+namespace Task1
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.Write("Enter X: ");
+            int x = Convert.ToInt32(Console.ReadLine());
+
+            Console.Write("Enter Y: ");
+            int y = Convert.ToInt32(Console.ReadLine());
+
+            Console.Write("Enter Z: ");
+            int z = Convert.ToInt32(Console.ReadLine());
+
+            var result1 = (x+y) * (Math.Pow(z,2)+1);
+            var result2 = (x % z - 1) * Math.Sqrt(y);

```

Comment:
@okuzmina Good format of expression! Please format like this expressions below and above